hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +37,43 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+    def test_team_access_works(self):
```

Comment:
I feel that the implementation of this test could be better.  Check in `hc/test.py`, you should see that it is stated clearly that bob already has team-access and thus he should be able to see Alice's channels.

The logic should there-fore be that once Alice creates a channel, then bob should see that channel after he logs in.